### PR TITLE
Support deployment of Klee-web

### PR DESCRIPTION
**Pull request by helicopter88**
- Update syntax in a variety of files
- Add DEPLOY.md with instructions for deployment

### CODE REVIEW

File: DEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* SSH into the VM from the Cloudstack console
+* Create a user named Ubuntu
+```bash
+adduser --ingroup sudo  ubuntu
+```
+* Login ubuntu and add your SSH key to ease the login process for ansible
+* Run `sudo visudo` and replace
+```%sudo ALL=NOPASSWD: ALL```
+with
+```%sudo (ALL=ALL) NOPASSWD: ALL```
```
Comment (ccadar) [inline DEPLOY.md]: I think the correct syntax is `%sudo ALL=(ALL) NOPASSWD: ALL`, right?

Comment (helicopter88) [inline DEPLOY.md]: Yes, I will fix it

File: provisioning/roles/web/tasks/flask.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@
 - name: Start supervisor service
   service: name=supervisor state=started
 
-# Create worker user
-- user: name=web shell=/bin/bash
+- name: Create web user
```
Comment (andronat) [inline provisioning/roles/web/tasks/flask.yml:18]: Where do we use flask?

Comment (andronat) [inline provisioning/roles/web/tasks/flask.yml:18]: Can you also rename this file to something more relevant?

**Comment by helicopter88** [comment]
Done

**Pull request merged**